vagrant/Vagrantfile

```diff
@@ -0,0 +1,139 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'socket'
+
+hostname = Socket.gethostname
+localmachineip = IPSocket.getaddress(Socket.gethostname)
+puts %Q{ This machine has the IP '#{localmachineip} and host name '#{hostname}'}
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = '2'
+
+# Validate required plugins
+REQUIRED_PLUGINS = %w(vagrant-registration vagrant-hostmanager landrush)
```

Comment:
probably want to limit vagrant-registration to just enterprise deployments, assuming origin deployments would be CentOS/Fedora only.